packages/apidom-ns-workflows-1/src/elements/WorkflowsSpecification1.ts

```diff
@@ -26,6 +26,14 @@ class WorkflowsSpecification1 extends ObjectElement {
   set info(info: InfoElement | undefined) {
     this.set('info', info);
   }
+
+  get sourceDescriptions(): ArrayElement | undefined {
```

Comment:
We ca have more specific `NCE` for this list of source description, let's use it.

```suggestion
  get sourceDescriptions(): SourceDescriptionsElement | undefined) {
```

Reply:
Wasn't it possible to use `SourceDescriptionsElement` here?

Reply:
All right, let's disregard it. Other namespaces are not doing this.